opentelemetry/chart/alerts/collector-alerts.yaml

```
@@ -0,0 +1,41 @@
+groups:
+  - name: collector-alerts
+    rules:
+  {{- if not (.Values.prometheusRules.disabled.FilelogRefusedLogs | default false) }}
+      - alert: FilelogRefusedLogs
+        expr: sum(rate(otelcol_receiver_refused_log_records_total{receiver=~"filelog"}[1m])) > 0
+        for: 5m
+        labels:
+          {{- include "plugin.additionalRuleLabels" . | nindent 6 }}
+          severity: warning
+          runbook_url: https://github.com/cloudoperators/greenhouse-extensions/tree/main/opentelemetry/playbooks/FilelogRefusedLogs.md
```

I would change the order here. This can led to helm template issues. The template goes to the end:
```suggestion
          severity: warning
          runbook_url: https://github.com/cloudoperators/greenhouse-extensions/tree/main/opentelemetry/playbooks/FilelogRefusedLogs.md
           {{- include "plugin.additionalRuleLabels" . | nindent 6 }}
```